src/main/java/application/rest/v1/ActionsEndpoint.java

```diff
@@ -139,8 +140,15 @@ public Response resolve(@Pattern(regexp = NAME_PATTERN_ONE_OR_MORE) @PathParam("
             ResponseBuilder builder = Response.ok(new ActionSubstitutionResolverResponse(resolve(client, name, kind, namespace, pattern)).getJSON());          
             return builder.build();
         }
-        catch (IOException | ApiException e) {
-            return Response.status(getResponseCode(e)).entity(getStatusMessageAsJSON(e)).build();
+        catch (IOException | ApiException | PatternException e) {
+            String msg = null;
+            if (e instanceof PatternException) 
```

Comment:
Where should the details of the return json structure be externally documented?

Reply:
Should it be documented in API design document?